dubbo-monitor/dubbo-monitor-api/src/main/java/org/apache/dubbo/monitor/support/CallbackMonitorFilter.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dubbo.monitor.support;
+
+import org.apache.dubbo.common.extension.Activate;
+
+import static org.apache.dubbo.common.constants.CommonConstants.CALLBACK;
+
+@Activate(group = CALLBACK)
+public class CallbackMonitorFilter extends MonitorFilter {
```

Comment:
Why add this class?

Reply:
`CallbackMonitorFilter` is the same as `MonitorClusterFilter`, because `MonitorClusterFilter` is a `ClusterFilter` that will not be loaded during a callback. Therefore, it is recommended to support the `MonitorFilter` logic on the callback side during a call to be compatible with `Skywalking`  tracking.

Reply:
@AlbumenJ  What is your point of view, delete it?

Reply:
This seem a little bad smell.

Reply:
I'm not sure if the bad smell is a naming issue, I have changed it to 'MonitorCallbackFilter'.
If there is any need to modify or delete, please let me know. 
@AlbumenJ

Reply:
What I mean by bad smell is that there are codes in Dubbo that Dubbo does not need to use

Reply:
Okay, it has been deleted now, users can add it themselves.